GUI/battle_menu.py

```diff
@@ -10,6 +10,8 @@
 
 combat_manager = CombatManager()
 
+COLUMN_MAX = 4
```

Comment:
Having it here is fine, as long as we don't add more stuff to this file. Otherwise I would put it inside class BattleMenu (on the class itself, not in the `__init__`).